test/suites/image_auto_update.sh

```diff
@@ -46,20 +46,15 @@ test_image_auto_update() {
   #
   # XXX: Since the auto-update logic runs asynchronously we need to wait
   #      a little bit before it actually completes.
-  retries=60
```

Comment:
Lets re-introduce this one but with retries=600, that will give us about 20min timeout which should be plenty even for the slowest of all ceph.

Reply:
I like your optimism :)